DigitalLearningSolutions.Web/ViewModels/LearningMenu/SectionCardViewModel.cs

```diff
@@ -0,0 +1,23 @@
+﻿namespace DigitalLearningSolutions.Web.ViewModels.LearningMenu
+{
+    using DigitalLearningSolutions.Data.Models.CourseContent;
+
+    public class SectionCardViewModel
+    {
+        public readonly string Title;
+        public readonly bool HasLearning;
+        public readonly double PercentComplete;
+
+        public SectionCardViewModel(CourseSection section)
+        {
+            Title = section.Title;
+            HasLearning = section.HasLearning;
+            PercentComplete = section.PercentComplete;
+        }
+
+        public string GetPercentComplete()
```

Comment:
Instead of having this as method you could store the percent complete string which you can create in the constructor. Not very important, but it means you don't need to store more fields than you need 🙂